regression/last_dev.py

```diff
@@ -91,18 +91,25 @@ def dev_scripts(script_pth, output_file_pth):
                     if log_data == "ai85tinierssdface":
                         continue
 
-                    temp[i+1] = str(config[log_data][log_model]["epoch"])
+                    try:
+                        temp[i+1] = str(config[log_data][log_model]["epoch"])
+                    except KeyError:
+                        # Handle the KeyError by assigning a dummy value
+                        temp[i+1] = "10"
 
                     if '--deterministic' not in temp:
                         temp.insert(-1, '--deterministic')
 
                     temp.insert(-1, '--name ' + log_name)
 
-                    data_name = temp[k+1]
-                    if data_name in config and "datapath" in config[data_name]:
+                    try:
                         path_data = config[log_data]["datapath"]
-                        temp.insert(-1, '--data ' + path_data)
+                        temp[i+1] = str(config[log_data][log_model]["epoch"])
+                    except KeyError:
+                        # Handle the KeyError by assigning a dummy value
+                        path_data = "/data_ssd"
```

Comment:
same comment with the previous one about the exception

Reply:
Handled with a warnings message.